src/Reflection/ReflectionType.php

```diff
@@ -73,14 +67,6 @@ public static function createFromNode(
         return new ReflectionUnionType($reflector, $owner, new UnionType($types));
     }
 
-    /**
-     * @internal
-     */
-    public function getOwner(): ReflectionParameter|ReflectionMethod|ReflectionFunction|ReflectionEnum|ReflectionProperty
```

Comment:
Hmm, removing this means no longer being able to run some checks in `roave/backwards-compatibility-check` 🤔

Reply:
I can also remove the `@internal` flag :)

Reply:
I'm not saying we shouldn't do it, I'm saying that we need some sort of approach to how to deal with it downstream :D 

Practically, if we move reflection stuff to be more and more immutable, we should totally do this.

Reply:
Do you really use this method in `roave/backwards-compatibility-check`? I'm not able to find it. Didn't you mean https://github.com/Roave/BetterReflection/blob/5.7.x/src/Reflection/ReflectionNamedType.php#L65 ?

Reply:
History of the method: https://github.com/Roave/BetterReflection/pull/975

Reply:
Hmm, perhaps we can try this with the other lib, as I may already have worked around it 🤷

Reply:
Checked again: LGTM, let's roll with it :)